### PR TITLE
add solution for day 27/11/24

### PR DESCRIPTION
**Pull request by lcaloi2904**
## Summary by CodeRabbit

- **Tests**
	- Introduced a new test case for the web application, ensuring key elements are visible and interactable for users.

### CODE REVIEW

File: daily-challenges/2024-11/27/lcaloi.29.spec.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { expect, Locator, test } from '@playwright/test';
+
+test('solution for day 27/11/24', async ({ page }) => {
+    await page.goto('https://material.playwrightvn.com/');
+    await expect(page.locator(`a[href='games/003-vi-tinh-tu.html']`)).toBeVisible();
+    await page.click(`a[href='games/003-vi-tinh-tu.html']`);
+    const targetName = await page.locator(`span#target-name`).textContent();
```
Comment (minhphong306) [inline daily-challenges/2024-11/27/lcaloi.29.spec.ts:7]: Ko dùng asertion ntn nha. Dùng web first assertion

File: daily-challenges/2024-11/27/lcaloi.29.spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    expect(targetName).toBeTruthy();
+    const mysteryBoxTarget: Locator = page.locator(`//*[@class='constellation-name' and text()='${targetName}']`);
+    page.on('dialog', async dialog => {
+        await dialog.accept();
+    });
+    await mysteryBoxTarget.click();
```
Comment (minhphong306) [inline daily-challenges/2024-11/27/lcaloi.29.spec.ts:13]: Thiếu assertion cho alert (chọn đúng vì tinh tú)

**Review by minhphong306 — CHANGES_REQUESTED** [review]

**Review by ducquaned — APPROVED** [review]
LGTM

**Review by minhphong306 — APPROVED** [review]
LGTM

**Pull request merged**